packages/server-wallet/src/protocols/channel-closer.ts

```diff
@@ -39,142 +38,137 @@ export class ChannelCloser {
   public async crank(objective: DBCloseChannelObjective, response: WalletResponse): Promise<void> {
     const channelToLock = objective.data.targetChannelId;
 
-    let attemptAnotherProtocolStep = true;
+    await this.store.lockApp(channelToLock, async tx => {
+      const channel = await this.store.getChannel(channelToLock, tx);
```

Comment:
I might be wrong, but isn't `channelToLock` already this channel?

Reply:
Thanks for pointing this out. The logic is changed in 39151e5a9.

My reasoning for using `store.getChannel` was that the method joins the channel row with the `ChainServiceRequests` table https://github.com/statechannels/statechannels/blob/e7ff05aef6e876d8e872137e3d2ceeb23aedfece/packages/server-wallet/src/wallet/store.ts#L250.

That's probably overkill. What was confusing me is that different channel database read methods join differring tables with the channels table